src/frequenz/dispatch/_dispatcher.py

```diff
@@ -213,6 +213,17 @@ async def start(self) -> None:
         """Start the actor."""
         self._actor.start()
 
+    async def resend_current_running_states(self, dispatch_type: str) -> None:
+        """Resend the current running states of all dispatches of a given type.
+
+        Warning: Usually you don't need to call this method. It is only useful
+        when you need to recover the current running state of your actor.
```

Comment:
This will highlight the warning properly:

```suggestion
        Warning:
            Usually you don't need to call this method. It is only useful
            when you need to recover the current running state of your actor.
```